DevCycle.SDK.Server.Local/Api/EventQueue.cs

```diff
@@ -153,24 +157,71 @@ public virtual void QueueEvent(DevCyclePopulatedUser user, DevCycleEvent @event,
                 logger.Log(LogLevel.Error, "Failed to queue an event. Events in queue exceeded the max");
                 return;
             }
+
             localBucketing.QueueEvent(
-                sdkKey, 
-                JsonConvert.SerializeObject(user), 
+                sdkKey,
+                JsonConvert.SerializeObject(user),
                 JsonConvert.SerializeObject(@event)
-                );
+            );
             logger.LogDebug("{Event} queued successfully", @event);
         }
 
+        public virtual void QueueSDKConfigEvent(RestRequest request, RestResponse response)
+        {
+            var popU = new DevCyclePopulatedUser(
+                new DevCycleUser(userId: $"{localBucketing.ClientUUID}@{Dns.GetHostName()}"));
+            QueueEvent(popU, new DevCycleEvent(
+                type: EventTypes.sdkConfig,
+                target: request.Resource,
+                value: -1,
```

Comment:
any way to get the response time in ms here?

Reply:
Not without wrapping our own timer - which isn't recommended